integration-cli/docker_cli_daemon_test.go

```diff
@@ -2070,3 +2070,20 @@ func (s *DockerDaemonSuite) TestRunLinksChanged(c *check.C) {
 	c.Assert(err, check.NotNil, check.Commentf(out))
 	c.Assert(out, check.Not(checker.Contains), "1 packets transmitted, 1 packets received")
 }
+
+func (s *DockerDaemonSuite) TestDaemonUngracefulShutdownCleanUpRootfsOfOverlay(c *check.C) {
```

Comment:
I don't understand how it works without it if it requires `/` to be mounted as overlay. It won't be run on CI without `DOCKER_GRAPHDRIVER=overlay`. Also, it did not test that there are no leftovers. Only that container was removed successfully.